.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "evm-semantics"]
 	path = evm-semantics
-	url = https://github.com/mrchico/evm-semantics
+	url = https://github.com/dapphub/evm-semantics
```

Comment:
Why not point it to upstream?

Reply:
we have a couple of small patches applied on top.